tests/cases/collection_test.py

```diff
@@ -404,3 +404,20 @@ def testMissingAclRefs(self):
         # Changes should have been saved to the database
         coll = collModel.load(coll['_id'], force=True)
         self.assertEqual(acl, coll['access'])
+
+    def testCollectionsDetails(self):
```

Comment:
New test cases should be implemented in pytest rather than the old unittest suite. Could you move this case over there?

Reply:
Ah, I didn't know that. I'll move it.

Reply:
Hm, I think I didn't find much-existing pytests to learn from. I am not so familar to move this test with a new test framework. Is that ok to have it here this time? If not, I can ask someone to help me take a look offline.

Reply:
I just pushed a commit to this branch that moves the test case to pytest. I wanted to parametrize over two fixtures, so had to hack that a bit using another fixture, due to limitations of pytest. There may be a more clever way to do it, but I couldn't figure out how to essentially pass a fixture as a parametrized argument.

Reply:
Thank you! I'll merge this now.